components/polygon/actions/get-company-financials/get-company-financials.mjs

```diff
@@ -0,0 +1,84 @@
+import {
+  SORT_HISTORICAL_OPTIONS,
+  SORT_OPTIONS, TIMEFRAME_OPTIONS,
+} from "../../common/constants.mjs";
+import polygon from "../../polygon.app.mjs";
+
+export default {
+  key: "polygon-get-company-financials",
+  name: "Get Company Financials",
+  description: "Retrieves financial details for a specific company by stock ticker. [See the documentation](https://polygon.io/docs/stocks/get_v3_reference_financials).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    polygon,
+    stockTicker: {
+      propDefinition: [
+        polygon,
+        "stockTicker",
+      ],
+    },
+    filingDate: {
+      type: "string",
+      label: "Filing Date",
+      description: `Query by the date when the filing with financials data was filed in **YYYY-MM-DD** format.
+                  \nBest used when querying over date ranges to find financials based on filings that happen in a time period.
+                  \nExamples:
+                  \nTo get financials based on filings that have happened after January 1, 2009 use the query param filing_date.gte=2009-01-01
+                  \nTo get financials based on filings that happened in the year 2009 use the query params filing_date.gte=2009-01-01&filing_date.lt=2010-01-01`,
+      optional: true,
+    },
```

Comment:
Since the component doesn't include any way to use `.gte` or `.lt` in the prop, I think that part should be left out of the description.
```suggestion
    filingDate: {
      type: "string",
      label: "Filing Date",
      description: "Query by the date when the filing with financials data was filed in **YYYY-MM-DD** format",
      optional: true,
    },
```